app/views/books/show.html.erb

```diff
@@ -13,7 +13,23 @@
       <img  src="/images/progit2.png" width="118" height="157"/>
       <div style="light">2nd Edition (2014)</div>
       <div class="switch"><a href="/book/<%= @book.code %>/v1">Switch to 1st Edition</a></div>
-    </div>
+      <div class="ebook-download">
```

Comment:
also, maybe we should only display this "Download Ebook" block if there's at least one to download? On my local version (probably because I didn't generate the book) I get the section but with no content. But maybe that case doesn't happen on the live app..?

Reply:
The case happens in fact. Up to now only English and French provide ebooks. So the whole "Download Ebook" section should disappear completely if no form of ebook is available.